crates/rbuilder/src/backtest/mod.rs

```diff
@@ -59,16 +59,22 @@ pub struct OrdersWithTimestamp {
     pub sim_value: Option<SimValue>,
 }
 
+/// Historic data for a block.
+/// Used for backtesting.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct BlockData {
     pub block_number: u64,
+    /// Info for landed block
```

Comment:
unclear, is this in reference to the next parameter only? then this should be about whatever this param is more specifically